src/app/dev/DevToys.Tools/Helpers/JsonHelper.cs

```diff
@@ -28,7 +28,8 @@ internal static bool IsValid(string? input, ILogger logger)
 
         try
         {
-            var jtoken = JToken.Parse(input);
+            using JsonReader reader = new JsonTextReader(new StringReader(input));
+            JToken? jtoken = await JToken.LoadAsync(reader, settings: null, cancellationToken);
```

Comment:
Changed this method so we can pass the CancellationToken of SmartDetection (which cancels within 2 seconds or if the clipboard content changed)

Reply:
We could use a ValueTask here.